src/service/ampdoc-impl.js

```diff
@@ -727,6 +727,14 @@ export class AmpDoc {
     return devAssert(this.visibilityState_);
   }
 
+  /**
+   * Whether the AMP document currently being previewed.
```

Comment:
currently is being previewed?

Reply:
I'll fix the comment in another PR, thanks for the review!